controlpanel/api/models/app.py

```diff
@@ -51,16 +51,23 @@ def customers(self):
     def add_customers(self, emails):
```

Comment:
Wouldn't you say that these belongs in `cluster.App` rather than in the model as they change state outside the CP itself.

Reply:
Sure, but that wasn't in the scope of this PR

Reply:
I realize I often do things that are outside the scope of a single PR 😅

Reply:
ahaha good point, it was one of my usual "thinking aloud" comments as it was on code you didn't touch 😆